### DISCUSSION TITLE
Using SHAP plot images need permision grant?

### DISCUSSION THREAD

**Discussion by amirhkr** [shap/shap#3184] [General]
Hi all

I would like to use two plot (force plot and beeswarm) displayed in the Github repo (https://github.com/shap/shap) in my PhD thesis. Do I need to get permission from the authors or it's ok if I use them? I read the MIT licence and it seems like it ok to use them. Can you confirm this?

**Comment by znacer** [comment]
Hi,
I had the same woried as yours. The MIT license applied to "this software and associated documentation files".
Those plots are part of the documentation. Hence there is no limitation to use them, according to the license.
